index.html

```diff
@@ -278,6 +293,7 @@ <h4>Skills</h4>
         
         <img src="images/contact-form-comp.svg" class="contact-img" alt="contact-form-img">
 
+        <div class="line"></div>
 
         <ul class="handle-icon set-icons">
             <li><a href=""><img class="svg" src="images/twitter.svg" alt="twitter logo"></a></li>
```

Comment:
- [ ] `required`
- [ ] kindly make the social icons as same as the figma template. why? it's important that you pay attention to all the details in this template and future templates because that will make you stand out from other developers. see screenshots below
-  `figma template`
![Screenshot from 2022-07-08 23-11-39](https://user-images.githubusercontent.com/66230306/178102921-f4791b9f-e1de-4d36-81ab-515db3a02401.png)

-  `your template`
![Screenshot from 2022-07-08 23-12-03](https://user-images.githubusercontent.com/66230306/178102952-f7c6ff3d-3649-44db-83de-6d9f54df9854.png)

Reply:
I have follow the instruction given and implemented this changes.